variables.tf

```diff
@@ -19,13 +19,19 @@ variable "vpc_id" {
   type        = string
 }
 
+variable "subnet_id" {
+  description = "Subnet used for the runner and executors."
+  type        = string
+  default     = ""
```

Comment:
Setting the default value to `""` won't work with the conditon you use like `var.subnet_id ? .....` The conditon should be evaluate to a boolean expression.